### PR TITLE
Fix - delete table bug

### PR DESCRIPTION
**Pull request by pranavxc**
Block table delete action if there is a relation column present.

re #1861

### CODE REVIEW

**Review by wingkwong — APPROVED** [review]
Verified

image

**Comment by o1lab** [comment]
We can improve the messaging.

It should something along : 
Unable to delete tables because of the following. 
- Column1 is a LinkToAnotherRecord of Table1 
- Column2 is a LinkToAnotherRecord of Table2

**Comment by pranavxc** [comment]
@o1lab : sample

Screenshot 2022-04-27 at 3 00 08 PM

File: packages/nc-gui/components/project/table.vue [hunk 1]
```diff
@@ -109,12 +109,14 @@ export default {
           const relationColumns = meta.columns.filter(c => c.uidt === UITypes.LinkToAnotherRecord)
 
           if (relationColumns.length) {
-            const referredTables = await Promise.all(relationColumns.map(async(c) => {
+            const refColMsgs = await Promise.all(relationColumns.map(async(c, i) => {
               const refMeta = await this.$store.dispatch('meta/ActLoadMeta', { id: c.colOptions.fk_related_model_id })
-              return (refMeta && refMeta.title) || c.title
+              return `${i + 1}. ${c.title} is a LinkToAnotherRecord of a ${(refMeta && refMeta.title) || c.title}`
```
Comment (wingkwong) [inline packages/nc-gui/components/project/table.vue:114]: I think `a` in `of a ${(refMeta && refMeta.title) || c.title}` is unncessary.

image

**Pull request merged**